### DISCUSSION TITLE
建议deepin-terminal分开中英文字体字号设置

### DISCUSSION THREAD

**Discussion by liwl1991** [linuxdeepin/developer-center#5596] [Features Request & Ideas | 特性请求 & 头脑风暴]
参考xshell，将非亚洲字体和亚洲字体分开设置，包括字体，字号，这样会美观一些。
当前版本特性中，中英文使用同一套字体，更换字体后，大小，宽度方面不太协调

**Comment by liwl1991** [comment]
@meiyixiang 
@ArchieMeng

**Comment by meiyixiang** [comment]
我研究一下

**Comment by ArchieMeng** [comment]
我忽然想起以前 @wineee 提到过，终端的emoji显示有问题，换个字体显示就好了。所以，拓展开来说， 也许需要分中文/英文/emoji等，这样分开来设置字体类型和大小。

**Reply by BLumia** [reply]
这时候就又要搬出来 https://bugreports.qt.io/browse/QTBUG-85744 了（手动 :doge: ）

**Reply by ArchieMeng** [reply]
草
